doc/classes/Input.xml

```diff
@@ -205,6 +226,25 @@
 				Returns the mouse mode. See the constants for more information.
 			</description>
 		</method>
+		<method name="get_vector" qualifiers="const">
+			<return type="Vector2">
+			</return>
+			<argument index="0" name="negative_x" type="StringName">
+			</argument>
+			<argument index="1" name="positive_x" type="StringName">
+			</argument>
+			<argument index="2" name="negative_y" type="StringName">
+			</argument>
+			<argument index="3" name="positive_y" type="StringName">
+			</argument>
+			<argument index="4" name="deadzone" type="float" default="-1.0">
+			</argument>
+			<description>
+				Similar to writing [code]Vector2(Input.get_action_strength("positive_x") - Input.get_action_strength("negative_x"), Input.get_action_strength("positive_y") - Input.get_action_strength("negative_y")).clamped(1)[/code]
```

Comment:
I would not say it is similar, as with this solution the deadzone is a cross and not a circle. With that wording, it is unclear that there is a significant difference between the two solutions.

Reply:
But aside from that, the PR is good to me.